Dockerfile

```diff
@@ -56,9 +56,9 @@ RUN export DEBIAN_FRONTEND='noninteractive' && \
       /usr/bin/c++ c++ /usr/bin/clang++-${CLANG_VERSION} 100 && \
     update-alternatives --install \
       /usr/bin/python python /usr/bin/python3 100 && \
-    curl https://sh.rustup.rs -sSf > rs.sh && \
-    chmod +x rs.sh && \
-    ./rs.sh -y && \
+    curl -sSf -L https://sh.rustup.rs -o /tmp/rs.sh && \
```

Comment:
in THEORY this can be 
```
curl -sSf -L https://sh.rustup.rs | sh -- -y
```
But I couldnt get it to work

Reply:
Weird. I'll give it a try and see if I can get that to work